docs/conf.py

```diff
@@ -215,6 +215,9 @@
     # Logos. Location is relative to _static folder.
     "light_logo": "logo1.svg",  # Logo for light mode
     "dark_logo": "logo1.svg",  # Logo for dark mode
+    "source_repository": "https://github.com/emdgroup/baybe/",
+    "source_branch": "main",
+    "source_directory": "docs/",
 }
 
 # Ignored links for linkcheck
```

Comment:
tried it locally, this does not seem to work for autodocumentation, eg:
![image](https://github.com/emdgroup/baybe/assets/17951239/8c99f2ff-7c0c-480b-a2dd-026f0cef7b55)

@AdrianSosic @AVHopp do we care about that? It sill works for user guides and examples and overall I think its a good suggestion

Reply:
If we can fix this such that it works for more than just the files that are available as .md files, then I do not see why we should not include this button. However, I also get 404 when building this locally. We would first need to investigate whether this button works as expected when being deployed on github pages.

Reply:
Hi @sgbaird, thanks a lot for the suggestion, I think having the button is a very good idea. But as long as we get the 404 errors for most pages, I think we cannot include it. Do you have any clue if this is configurable, i.e. can we restrict the appearance of the button to certain pages? If not, I would perhaps open an issue on the furo repo and ask for help there...

Reply:
Thank you for testing this out, I probably should have done a local build first. I will explore a bit more, but no worries if this one doesn't go through. I wasn't expecting the 404 errors 😅

Reply:
The best way to test the documentation would be to not only build it locally, but also build it as the github pages for your fork. This should ensure that 404 errors and stuff like that do not just happen because the local structure of the converted files is different to the way they are organized on github 😄